src/coreclr/binder/assemblyname.cpp

```diff
@@ -141,7 +141,7 @@ namespace BINDER_SPACE
         }
 
         // Set public key and/or public key token (if we have it)
-        if (pvPublicKeyToken && dwPublicKeyToken)
+        if (dwPublicKeyToken && pvPublicKeyToken)
```

Comment:
`pvPublicKeyToken` may be uninitialized, but then `pvPublicKeyToken` is `0` so there is no real issue here.
Swapping the order to make valgrind happy.